### ISSUE TITLE
Tri "Statut Fantoir" inopérant

### ISSUE THREAD

**Issue by deuzeffe** [osm-fr/osm-vs-fantoir#91]
Hello,

Ça https://github.com/osm-fr/osm-vs-fantoir/issues/89 est une super "feature". Marchi bocou !

Un bémol : le tri est inopérant sur la colonne "statut Fantoir" :/

(mais c'est bien quand même ^^)

**Comment by gendy54** [comment]
Oui, il faut un crack de javascript pour ca. SI tu en as un sous la main ;)

**Comment by deuzeffe** [comment]
@vdct c'est encore d’actualité ou je peux fermer ?
(/me a un peu perdu le fil)

**Comment by vdct** [comment]
@deuzeffe Statut inchangé là dessus, et je manque de connaissances pour m'y plonger. Si quelqu'un a envie d'y regarder faut pas hésiter....
